src/d3d8/d3d8_format.h

```diff
@@ -38,7 +49,7 @@ namespace dxvk {
       case D3DFMT_V8U8:
       case D3DFMT_L6V5U5:
       case D3DFMT_D16_LOCKABLE:
-      case D3DFMT_D15S1:
+      //case D3DFMT_D15S1:
```

Comment:
We should still return the correct stride for unsupported formats.

Reply:
Alright, will leave these as they were.